README.md

```diff
@@ -277,7 +277,7 @@ custom:
       barExceptions:
         metric: barExceptions
         threshold: 0
-        statistic: Minimum
+        statistic: Sum
```

Comment:
I agree that **Minimum** is an incorrect statistic to use here, but i would argue that **Maximum** is more appropriate by [definition](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Statistics-definitions.html):
`Maximum is the highest value observed during the specified period.`

So if we want to trigger an alert when there are more pattern matches than a given threshold, we need to look at the Maximum number of matches observed during the specified period.

Its counter-intuitive, but thats what i arrived at by trial and error